src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -306,18 +308,38 @@ using namespace Microsoft::Console::Interactivity;
             switch (level)
             {
             case ApiLevel::Win32:
+            {
                 pseudoClass.lpszClassName = PSEUDO_WINDOW_CLASS;
-                pseudoClass.lpfnWndProc = DefWindowProc;
+                pseudoClass.lpfnWndProc = s_PseudoWindowProc;
                 RegisterClass(&pseudoClass);
+
+                const auto windowStyle = WS_OVERLAPPEDWINDOW;
+                const auto exStyles = WS_EX_TOOLWINDOW | WS_EX_TRANSPARENT | WS_EX_LAYERED;
                 // Attempt to create window
-                hwnd = CreateWindowExW(
-                    0, PSEUDO_WINDOW_CLASS, nullptr, WS_OVERLAPPEDWINDOW, 0, 0, 0, 0, HWND_DESKTOP, nullptr, nullptr, nullptr);
+                hwnd = CreateWindowExW(exStyles,
+                                       PSEUDO_WINDOW_CLASS,
+                                       nullptr,
+                                       windowStyle, //WS_CHILD, //WS_OVERLAPPEDWINDOW,
+                                       0,
+                                       0,
+                                       0,
+                                       0,
+                                       HWND_DESKTOP, // owner
+                                       nullptr,
+                                       nullptr,
+                                       this);
+
                 if (hwnd == nullptr)
                 {
                     DWORD const gle = GetLastError();
                     status = NTSTATUS_FROM_WIN32(gle);
                 }
+
+                /*BOOL const cloak = TRUE;
+                DwmSetWindowAttribute(hwnd, DWMWA_CLOAK, &cloak, sizeof(cloak));*/
+

```

Comment:
Also trim. We won't use this here. It'll be added later to hide dialogs if we pan away from the tab.